content/en/agent/faq/linux-key-rotation-2024.md

```diff
@@ -0,0 +1,201 @@
+
+Customers using Datadog's RPM or DEB packages might require a manual action to import the new key on their systems to install or upgrade the Agent packages after the rotation takes place.
+
+<div class="alert alert-info">
+<strong>Note</strong>: This DOES NOT affect the functionality of already running Agents. It only limits the ability to install or upgrade to a newer version of the Agent. Also, this doesn't affect Dockerized Linux Agents, Windows, or macOS Agents.
+</div>
+
+## Install methods that automatically trust the new GPG key
+
+Your host automatically trusts the new key (no further action is required) if you're using one of the following install methods:
+
+- [Agent install script][5] v1.18.0+ (released Jun 27, 2023)
+- [Chef cookbook][6] v4.18.0+ (released Jul 26, 2023)
+- [Ansible role][7] v4.20.0+ (released Jul 18, 2023)
+- [Ansible collection][14] v5.0.0+ (released Jul 18, 2023)
+- [Puppet module][8] v3.21.0+ (released Jul 05, 2023)
+- [SaltStack formula][9] v3.6+ (released Aug 10, 2023)
+- [Heroku buildpack][10] v2.11+ (released Jun 15, 2023)
+- [Elastic Beanstalk][11] config templates updated as of Jun 27, 2023 or later (should contain `source: https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh`)
+- Containerized Agents (Docker/Kubernetes) for any version
+- Windows/MacOS Agents for any version
+
+Additionally, installing the DEB Agent v6.48.0+ or v7.48.0+ package through `apt` from the `apt.datadoghq.com` repository installs the [`datadog-signing-keys` package](#the-datadog-signing-keys-package) version 1.3.1. It automatically ensures that your host trusts the new key. If you have `datadog-signing-keys` version 1.3.1 or later installed, no further action is needed. Versions of `datadog-signing-keys` older than version 1.3.1 don't guarantee full preparedness for the key rotation.
+
+If you installed Observability Pipelines Worker or APM tracer libraries **using the above install methods**, they already come with the newest keys. No further action is required.
+
+If you are installing the DEB Agent package from a different repository or you are not using `apt` (or a similar tool that checks repo metadata signatures), your system doesn't need to know the Datadog signing keys. No further action is needed. However, you may benefit from the [`datadog-signing-keys` package](#the-datadog-signing-keys-package).
+
+If you're unsure if a host trusts the new signing key, you can [check](#check-if-a-host-trusts-the-new-gpg-key).
+
+For hosts running older versions of the install methods listed above or older versions of the DEB package, Datadog recommends updating the install method to the latest version. Alternatively Debian and Ubuntu users can update the Agent to version 7.48.0+. Otherwise, the key can be [manually updated](#manual-update).
```

Comment:
```suggestion
For hosts running older versions of the install methods listed above or older versions of the DEB package, Datadog recommends updating the install method to the latest version. Alternatively, Debian and Ubuntu users can update the Agent to version 7.48.0+. Otherwise, the key can be [manually updated](#manual-update).
```